docs/guides/cloudeventstarget.md

```diff
@@ -0,0 +1,57 @@
+# Exporting CloudEvents
+
+The Trigermesh `CloudEventsTarget` API object is used to export Cloudevents to external systems via HTTP.
+
+## Configuring a CloudEventsTarget Object
+
+The CloudEventsTarget accepts parameters to set the endpoint where CloudEvents will be sent and authentication. When succesfuly created it consumes and forwards to the configured an HTTP endpoint.
+
+### Configuring CloudEvents Endpoint
+
+The `spec.endpoint` parameter is a destination that points to an HTTP URL that will receive the ingested CloudEvents.
+
+!!! example "Using a reference"
+    ```yaml
+    spec:
+      endpoint: https://external.systen/mypath
+    ```
+
+### Configuring Credentials (Optional)
+
+If the external system requires [Basic Authentication](https://datatracker.ietf.org/doc/html/rfc2617) this component can be configured to use credentials by means of Kubernetes secrets.
+
+The credentials are defined under `spec.credentials.basicAuths`:
+
+!!! example "Credentials for 2 users"
```

Comment:
I don't understand why this says 'for 2 users'? it seems to be one user/pass only no?

Reply:
copy pasta :-1: